src/content/docs/style-guide/writing-docs/article-templates/whats-new-template.mdx

```diff
@@ -0,0 +1,343 @@
+* The following instructions tell you how to do most of this directly in the GitHub UI, but if you are familiar with Git, you can push your changes from your computer using the command line, GitHub CLI, GitHub Desktop, or related tools.
+
+## Prepare your “What’s new” files [#create-a-file]
+
+"What's new" posts consist of a Markdown file (`.md`) explaining the feature, along with optional image files.
+
+Here's how to prepare your post and images:
+
+1. In VS Code, create a new file by clicking  **File > New File**. 
+2. Copy this simple example post (CTRL + A) and paste it (CTRL + V) into the new, empty file. If you prefer to start with a template that has a wider-range of examples, you can copy the [detailed template](#detailed-template) instead.
+```
+---
+title: 'Control data ingest costs with ingest drill-down'
+summary: 'Analyze the data that you send to New Relic'
+releaseDate: '2021-06-30'
+learnMoreLink: 'https://docs.newrelic.com/docs/telemetry-data-platform/manage-data/manage-data-coming-new-relic/#facet-data-ingest'
+getStartedLink: 'https://one.newrelic.com/-/0gbRKzygzwE'
+---
+
+Inspect your data consumption in New Relic to better manage your data ingest and costs. Understand your data usage by each data source, and analyze it using your existing metadata, such as by source, team, and any other data sample attribute.
+
+To get started, go to the **Data management hub** and select the **Data ingestion chart**. Get detailed insights by data source by clicking on the colored data band at any point in time that you’re interested in. Add "facets" to analyze your data by selected attributes. [**Learn more**](https://docs.newrelic.com/docs/telemetry-data-platform/manage-data/manage-data-coming-new-relic/#facet-data-ingest) and [**get started**](https://one.newrelic.com/-/0gbRKzygzwE).
+
+![To find your data ingest usage, click the account dropdown, click Manage your data, then click Data ingestion.](./images/ingest_drill_down_1_of_2.png "A screenshot that shows how to find your data ingest consumption.")
+
+<figcaption>To find your data ingest consumption, click Manage your data, then click Data ingestion.</figcaption>
+
+![Choose a specific data source to drill down into what's contributing the most to your data ingest consumption.](./images/ingest_drill_down_2_of_2.png "A screenshot that shows the data ingest by data source dropdown.")
+
+<figcaption>Click the chart to see ingestion by data source.</figcaption>
+```
+3. Replace the example content with details from your post. Don't forget to change the front matter entries at the top!
+4. Save the file to a convenient working directory and give it a name with this format: `whats-new-MM-DD-descriptive-words.md`(for example, `whats-new-5-28-in-context-alerting.md`).
+5. Save the images you reference in your Markdown to the same working directory on your computer. 
+    * Be sure to remove spaces in your image file names. 
```

Comment:
@daynaslord, I clarified these sub-points like this:

    * Make sure you don't have any spaces in image file names. You can replace spaces with dashes or underscores like this: `automap_preview.png`. 
    * Make sure all your references to images in the Markdown file are prefixed with `./images/` like this: `./images/automap_preview.png`. This is because you'll eventually upload  them to the `images` folder in GitHub.